include/nuttx/usb/cdc.h

```diff
@@ -890,15 +930,15 @@ struct cdc_linecoding_s
 struct cdc_linestatus_s
 {
   uint8_t size[2];   /* wLength, Size of this structure, in bytes */
-  uint8_t ringer[4]; /* dwRingerBitmap, Ringer Configuration bitmap for this line */
+  uint8_t ringer[4]; /* dwRingerBitmap, Ringer Conf bitmap for this line */
```

Comment:
```suggestion
  uint8_t ringer[4]; /* dwRingerBitmap, Ringer Configuration bitmap for this
                              * line */
```

Conf is not an acceptable abbreviation for anything.